examples/complete/variables.tf

```
@@ -5,5 +5,15 @@ variable "region" {
 
 variable "parent_zone_name" {
   type        = string
-  description = "Parent zone name"
+  description = "Name of the hosted zone to contain this record (or specify `parent_zone_id`)"
+}
+
+variable "ns_record_ttl" {
+  type        = number
+  description = "The time to live (TTL) of the nameserver Route53 record, in seconds."
+}
+
+variable "soa_record_ttl" {
+  type        = number
+  description = "The time to live (TTL) of the start of authority Route53 record, in seconds."
```

Keep existing default for backwards compatibility

```suggestion
  description = "The time to live (TTL) of the start of authority Route53 record, in seconds."
  default     = "30"
```

---

Closing. See above.